pkg/logs/agent_test.go

```diff
@@ -0,0 +1,99 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package logs
+
+import (
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/logs/sender/mock"
```

Comment:
nits: this import should be grouped with other `github.com/DataDog/datadog-agent/pkg/logs/` imports

Reply:
fixed